go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/code-ready/machine-driver-libvirt
 
 require (
-	github.com/code-ready/machine v0.0.0-20190904103148-0fde37b5d95b
-	github.com/libvirt/libvirt-go v3.4.0+incompatible
+	github.com/code-ready/machine v0.0.0-20191122132905-c31e0b90623d
+	github.com/libvirt/libvirt-go v6.4.0+incompatible
```

Comment:
By default, rhel 8.2 has libvirt 4.5.0, the advanced-virt module has 6.0.0. v6.4.0 seems too new, especially as I'd prefer if we also support rhel 8.1 (no written rule for crc, but running on the latest 2 minor releases of LTS distros, and latest 2 major releases of faster moving distros is a rule I've seen used in other projects). 8.1 apparently has libvirt 4.5 and 5.6

Reply:
@guillaumerose libvirt version shouldn't be updated to latest one otherwise it will break in the older system which have libvirt version less that 6.4.0, it is better to have it 3.4 to support wide range of systems even minikube also have same https://github.com/kubernetes/minikube/blob/master/go.mod#L47

Reply:
https://github.com/code-ready/machine-driver-libvirt/commit/0e78a57f97ed81422597bcfa9901e1ddb44678bb#diff-37aff102a57d3d7b797f152915a6dc16 have some info around same.

Reply:
Thanks! I will stay on 3.4.